cogs/audio.py

```diff
@@ -1795,21 +1858,56 @@ def _clean_url(self, url):
 
         song_info = []
         for num, song in enumerate(tempqueue_song_list, 1):
+            str_duration = str(datetime.timedelta(seconds=song.duration))
             try:
-                song_info.append("{}. {.title}".format(num, song))
+                if song.title is None:
+                    song_info.append("**[{}]** {.webpage_url} ({})".format(num, song, str_duration))
+                else:
+                    song_info.append("**[{}]** {.title} ({})".format(num, song, str_duration))
             except AttributeError:
-                song_info.append("{}. {.webpage_url}".format(num, song))
+                song_info.append("**[{}]** {.webpage_url} ({})".format(num, song, str_duration))
 
         for num, song in enumerate(queue_song_list, len(song_info) + 1):
-            if num > 5:
+            str_duration = str(datetime.timedelta(seconds=song.duration))
+            if num > 10:
                 break
             try:
-                song_info.append("{}. {.title}".format(num, song))
+                if song.title is None:
+                    song_info.append("**[{}]** {.webpage_url} ({})".format(num, song, str_duration))
+                else:
+                    song_info.append("**[{}]** {.title} ({})".format(num, song, str_duration))
             except AttributeError:
-                song_info.append("{}. {.webpage_url}".format(num, song))
+                song_info.append("**[{}]** {.webpage_url} ({})".format(num, song, str_duration))
+
         msg += "\n***Next up:***\n" + "\n".join(song_info)
+        em.description = msg.replace('None', '-')
+        more_songs = len(self.queue[server.id][QueueKey.QUEUE]) - 10
+        if more_songs > 0:
+            em.set_footer(text="And {} more songs...".format(more_songs))
+        await self.bot.say(embed=em)
+
+    def _draw_play(self, song):
+        song_start_time = song.song_start_time
+        total_time = datetime.timedelta(seconds=song.duration)
+        current_time = datetime.datetime.now()
+        elapsed_time = current_time - song_start_time
+        sections = 12
+        loc_time = round((elapsed_time/total_time) * sections)  # 10 sections
+
+        bar_char = '━'
+        seek_char = ':radio_button:'
+        play_char = ':arrow_forward:'
```

Comment:
Personally I prefer all emoji characters to be represented in code with `\N{}` notation, like this:
```py
bar_char = '\N{BOX DRAWINGS HEAVY HORIZONTAL}'
seek_char = '\N{RADIO BUTTON}'
play_char = '\N{BLACK RIGHT-POINTING TRIANGLE}'
```

Using raw emoji characters in code can lead to problems for people using a font in their text editor which doesn't support the character. As for the colon notation, I've it can lead to inconsistencies when used in various scenarios.